dessia_common/workflow/core.py

```diff
@@ -163,13 +183,16 @@ class Block(DessiaObject):
     _non_serializable_attributes = []
 
     def __init__(self, inputs: List[Variable], outputs: List[Variable],
-                 position: Tuple[float, float] = (0, 0), name: str = ''):
+                 position: Tuple[float, float] = None, name: str = ''):
```

Comment:
Tuple default assignment works perfectly fine as these are not mutables

Reply:
See the answer just above